src/ApiClient.js

```diff
@@ -411,7 +411,7 @@ class ApiClient {
   hostSettings() {
     return [
       {
-        url: 'https://api.usepatch.com',
+        url: 'https://{defaultHost}',
```

Comment:
Do you know what the cause of this diff is? Where is `hostSettings()` used?

Reply:
Not exactly sure but it seems like it would be a mistake - `defaultHost` is defined below this line.